### DISCUSSION TITLE
How to get PWA features

### DISCUSSION THREAD

**Discussion by yorqat** [emilk/egui#1892] [Q&A]
I want to know how to get this [PWA file system access feature](https://whatpwacando.today/file-system) with an egui app built on wasm32.

![image](https://user-images.githubusercontent.com/80141845/183017328-1b0ab6a0-42e2-441c-a5ab-5b5f5e952605.png)

**Comment by emilk** [comment]
I don't know much about PWA, but maybe check out https://github.com/emilk/eframe_template/ - it has a  PWA template (in https://github.com/emilk/eframe_template/tree/master/docs)
